glossary.yml

```diff
@@ -10454,4 +10460,11 @@
       of a binary classifier at different [classification](#classification) thresholds. The curve is 
       obtained by plotting the True Positive Rate (also known as Recall or [Sensitivity](#sensitivity))
       along the vertical axis and the False Positive Rate along the horizontal axis.
+  es:
+    term: "Curva ROC"
+    def: >
+      Una curva ROC (acrónimo de Receiver Operating Characteristic, o Característica Operativa del Receptor) 
+      es un gráfico que muestra el desempeño de un clasificador binario con diferentes umbrales de [classificación](#classification)
```

Comment:
"classificación" should be "clasificación"

"Sensitividad" should be "Sensibilidad"

"taza" should be "tasa"